neuraxle/base.py

```diff
@@ -205,6 +207,9 @@ def load_step(self, step: 'BaseStep', context: 'ExecutionContext') -> 'BaseStep'
         """
         raise NotImplementedError()
 
+    def get_saved_files(self, step: 'BaseStep', context: 'ExecutionContext') -> List[Tuple[str, str]]:
+        raise NotImplementedError()
```

Comment:
Can you explain this to me? I would have assumed that the files were always within some kind of folder. This would mean that someone could just load *everything* inside this folder. An I wrong? Is it possible to do that? I think it's possible to remove this method and have just big folders.

Reply:
e.g.: in a given subfolder, have everything dumped under another subfolder named something like `__dump__`. And then other subfolders next to the dump folder are folders of substeps in the case of a truncable or of a metastep